### ISSUE TITLE
URGENT need of assistance

### ISSUE THREAD

**Issue by einsrojas08** [nanopool/nanominer#211]
Hi I accidentally forgot to change the wallet name and it deposited my almost a year's worth of mining. Would you be kind to assist me on this?

**Comment by Daniel-1276** [comment]
What do you mean with "forget to **change** the wallet name" ? Do you mean you didn't specify any wallet address ? In general, funds go to the specified wallet and only the owner of private key is able for that specific wallet is able to transfer funds to any new address. So in case your funds are stuck in a wallet that does not belong to you, you are screwed.

**Comment by einsrojas08** [comment]
Is there a way to track or contact the owner of the wallet: 0xb2aaAbC10E82Fc1efA7eB7e8fdF49aA548B68fcE

Regards,
Einstein C. Rojas
Master of Science in Innovation and Business 2021
Aboitiz School of Innovation, Technology, and Entrepreneurship
[https://s3.amazonaws.com/aimmakati-cdn/aim_email_logo.jpg]

**Comment by Daniel-1276** [comment]
Not an official one ... you could try to sent a transaction with some [HEX/ASCII payload](https://ethereum.stackexchange.com/questions/2466/how-do-i-send-an-arbitary-message-to-an-ethereum-address), but it is unlikly that the receiving wallet will be able to read it as plain text. I mean, from which source did you get that address ? Was it an example of a youtube'r ? That's a common trap for newbees to just copy paste configuration files. This ETH wallet is holding around 550$ ... is that purely your mining outcome or did somebody already withdraw a lot of it ?

**Comment by einsrojas08** [comment]
Yes it was all from the mining using a 1060 and 1070 video card. That is what I am unsure it could be from the configuration file from the github link nanominer suggested.

Einstein C. Rojas
Master of Science in Innovation and Business 2021
Aboitiz School of Innovation, Technology, and Entrepreneurship
[https://s3.amazonaws.com/aimmakati-cdn/aim_email_logo.jpg]

**Comment by Daniel-1276** [comment]
Well just out of my own interest, you had your PC running for a year and didn't check your (own) wallet if the funds do arrive correctly? Well, can't believe it to be honest, but if true, I feel sorry for you. Please close this request, as developers cannot help in this case.
